### PR TITLE
PSP-2846 - GitHub action "CI-CD PIMS Test"

### PR DESCRIPTION
**Pull request by rafaelponcedeleon**

### CODE REVIEW

File: .github/workflows/ci-cd-pims-test.yml
Comment (devinleighsmith) [inline .github/workflows/ci-cd-pims-test.yml:11]: this seems incorrect. The destination for this build in jenkins is test

Comment (rafaelponcedeleon) [reply .github/workflows/ci-cd-pims-test.yml:11]: thanks for catching this. I'll update it

**Review by devinleighsmith — APPROVED** [review]
Approved, but would like @asanchezr to review this as well.

**Comment by rafaelponcedeleon** [comment]
I'll wait for @asanchezr to review before merging

**Comment by asanchezr** [comment]
Reviewing ATM

**Review by asanchezr — APPROVED** [review]
Looks good. Great work!

**Pull request merged**
